docs/src/main/asciidoc/building-native-image.adoc

```diff
@@ -424,6 +424,12 @@ If one of those extensions is present, then creating a container image for the n
 ----
 
 * `quarkus.native.container-build=true` allows for creating a Linux executable without GraalVM being installed (and is only necessary if you don't have GraalVM installed locally or your local operating system is not Linux)
+
+[NOTE]
+====
+If you're running a remote Docker daemon, you need to replace `quarkus.native.container-build=true` with `quarkus.native.remote-container-build=true`.
```

Comment:
This is already mentioned in https://github.com/quarkusio/quarkus/blob/b9ba708f477d2aec1873d4f4e041e061793e49a7/docs/src/main/asciidoc/building-native-image.adoc#creating-a-linux-executable-without-graalvm-installed (2nd tip).

I suggest adding a reference to that tip as it contains more details about why this is necessary.

Reply:
Thanks for your review, @zakkak.
I've added the suggested change.